README.md

```diff
@@ -10,17 +10,18 @@ To run the TypeScript examples:
 ```
 $ npm install -g aws-cdk
 $ npm install
-$ npm build
-$ cdk deploy
+$ npm run build
+$ cdk synth   // Optional; Creates the Cloudformation template
```

Comment:
I'd really really rather not have this line here. Even not with the annotation :)

Reply:
alright